sr_robot_launch/launch/sr_ur_arm_hand.launch

```diff
@@ -68,13 +68,17 @@
 
   <!-- SIMULATED ROBOTS -->
   <arg name="side" default="right"/>
-  <arg name="hand_version" default="E3M5"/>
-  <arg name="fingers" default="th,ff,mf,rf,lf"/>
-  <arg name="tip_sensors" default="th=pst,ff=pst,mf=pst,rf=pst,lf=pst"/>
+  <arg name="hand_type" default="hand_e"/>
```

Comment:
This is a launch file that will be called by the others. I am wondering if we really need the ifs here. maybe set the defaulsts to hand e with all and pst?

Reply:
I am setting only the defaults, if the arguments are set from a top-level launch file, the values within this one will be overridden. With defaults and the ifs, it's possible to launch only this as a standalone without the issue of type&version mismatch.